### DISCUSSION TITLE
Another apostate with an interesting paper

### DISCUSSION THREAD

**Discussion by budrap00** [a-cosmology-group/acg#228] [Cosmology Models]
https://www.academia.edu/39329821/Towards_a_More_Well_Founded_Cosmology?email_work_card=title

Han will appreciate this footnote:

>By ‘fictitious’, we mean ‘merely existing in theory, not in reality’.
In contrast, the so-called ‘fictitious forces’ are never fictitious in this
common sense, but rather in the opposite sense, which reflects a
theory-centred world view that is characteristic of theoretical physics.

**Comment by HanDeBruijn** [comment]
### Towards a More Well-Founded Cosmology

@ExpEarth , as quoted from the paper:
>Edwards [97] considered the possibility that the
energy that is lost by photons while they become redder
might remain present in gravitational form and flow into
potential wells. From these, the energy would eventually
be recycled. [ .. ]
[96] R. Edwards, Apeiron 5, 157 (1998).
[97] R. Edwards, Astrophys. Space Sci. 339, 13 (2012).

Are these references yours, Matt?

**Reply by ExpEarth** [reply]
Yes, Han.  That first one was my first paper in this field.  That idea expressed there is the very same one you said was impossible in the other thread.  But it makes total sense.  Just substitute some kind of photonic spacetime for 'potential wells'.

**Reply by budrap00** [reply]
>photonic spacetime

That sounds like a euphemism for electromagnetic radiation?

**Reply by ExpEarth** [reply]
Think of it as em radiation in a highly structured form.  The basic model I use now is similar to this one by Arto Annila, which I have mentioned before.  Arto has the view that everything is made up of photons and this sounds right to me.  I would add to Annila's picture that the filaments of paired photons (which he and I both call gravitons) are linked to masses.  With that configuration you can get to gravity.

![image](https://github.com/a-cosmology-group/acg/assets/136624758/a0b013a2-0988-442a-b093-39fd547cbca4)
`The paired-photon vacuum.  Vacuum spectral density, u, sums up from numerous rays of photons (blue-red waves) with a spectrum of energies, εi, about the average energy, kBT. The paired photons cannot be seen as light but are sensed as inertia and gravitation through their coupling to matter.  In contrast, the odd quanta (blue or red), distributed in-phase or antiphase among the paired rays, are seen as light and manifest as electromagnetism. Inset: The cumulative spectral density vs. energy (dashed line) (from Annila and Wikström [39]).`

**Reply by budrap00** [reply]
>The paired photons cannot be seen as light but are sensed as inertia and gravitation through their coupling to matter.

If you define gravitons (paired photons) as inherently undetectable except as inertia and gravitation which are the phenomenological effects you attempting to account for, then it seems to me you are just making a circular argument similar to those made for dark matter or substantival spacetime. Attributing observed effects to unobservable causes does not lead to a deepening of our scientific understanding.

OTOH, I agree with your general premise that inertial-gravitational effects are a consequence of matter-energy interactions, energy in this case being the aggregate of all the omnidirectionally sourced electromagnetic radiation streaming throughout the Cosmos.

**Reply by ExpEarth** [reply]
Well, we can't exactly 'see' what a photon is, even in the form of ordinary radiation.  All our knowledge about them comes from studying the effects they produce.  We can't really see behind the veil.

That said, the evidence for the paired-photon vacuum (PPV) is pretty substantial.  If we think of the potential well as being a concentration of this PPV in the vicinity of masses, and then suppose that this radiation also undergoes a Hubble redshift, same as ordinary light does, then we will see radiation gradually leaked out of these wells.  I describe this in that second paper Han mentioned.  Here is my strong evidence for this process:

![image](https://github.com/a-cosmology-group/acg/assets/136624758/ba0b1540-2519-44d8-acd3-785456e75ab2)

`Hubble luminosity at many scales.  On the horizontal axis are plotted the bolometric luminosities of representative white dwarfs (WD); neutron stars (NS); supermassive black holes (SMBH); the observable universe (U); and the excess heat emissions of Earth and Jupiter.  Their respective Hubble luminosities are on the vertical axis.  The solid line is the 1:1 correspondence.  The range of luminosities covers 35 orders of magnitude.  Adapted from [51].`

**Reply by HanDeBruijn** [reply]
Concerning the _Hubble luminosity at many scales._, I've found back the relevant [thread](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7138316) and the relevant [paper](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/HubbleRedshift.pdf).
However, on a double logarithmic scale like the above, it seems not so difficult to manage that _anything appears linear_.

**Comment by ExpEarth** [comment]
It's remarkably linear, Han.  The equation is really simple, _L = -UH_ , where _U_ is the internal gravitational potential energy and _H_ is the Hubble constant.  There are many unusual things about this.  The luminosities for white dwarfs and neutron stars should not be as high as they are.  People invent novel processes to explain it, but each one is ad hoc for each class of object.  For the SMBHs, for instance, the standard explanation is that their energies arise from energy released in accretion of matter.  But their luminosities are way too constant to be explained this way.  In some cases the Hubble luminosity prediction is only about 10 % of the actual observed luminosity, but that's still pretty close.  I did a paper on white dwarfs specifically, here: http://redshift.vif.com/JournalFiles/V15NO4PDF/V15N4EDW.pdf

What we really need is a way to test it in objects where other sources of heating are very unlikely.  I think small icy planets, moons or asteroids would be best, provided there are no significant tidal forces anywhere.  It would be hard to test this in the lab, as the output would be so small, but maybe not impossible.

**Reply by HanDeBruijn** [reply]
@ExpEarth. Again your theory has been inspiring for musings by myself. That's the good news.
The "bad" news is that I've been able now to debunk by own theory about the [CMB](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm). As follows.
Replace radius and mass of the Universe by the same quantities for e.g. the Sun. Assume the same mechanism as defining the CMB, namely
### Uniform CMB energy + Gravitational self energy = 0
Then we have, analogously
```
M := 1.989*10^30; # Mass of sun
R := 696265*10^3; # Radius of sun
G := 6.6743*10^(-11);
sigma :=  5.670374419*10^(-8);
L := G*M^2/(2*R); # Self Energy
c := 299792458;
Theta := (L/(4/3*Pi*R^3*4*sigma/c))^(1/4);
evalf(%);

          20,518,794.89 K
```
However, [Wikipedia](https://en.wikipedia.org/wiki/Stefan%E2%80%93Boltzmann_law#Temperature_of_stars) gives _much lower temperatures_ for the sun's surface, its inner body, even its corona. We must conclude that the basic assumption of our **theory** gives a result that is absurd . Calculated temperatures are orders of magnitude **wrong**.
This proves that one's theory can mimick any outcome as desired, even without adjusting any free parameters. And now give me some time to recover from the shocking discovery  :-(

**Reply by ExpEarth** [reply]
Your relationship won't work for stars, because the energy content of stars is dominated by fusion processes.  You have to look at other objects.  Where it really describes the situation quite well is for 'black holes', meaning, objects where the mass is inside or at the Schwarzschild radius.  Assuming the universe is such an object, we find that the energy of the CMB matches the gravitational energy for a CMB temperature of about 27-29 K.  That is much greater than the local temperature of 2.7, but it fits with an average cosmic CMB temperature assuming a static universe where the Hubble radius corresponds to the Schwarzschild radius (in some sense).

**Reply by HanDeBruijn** [reply]
@ExpEarth about my CMB model:
>Where it really describes the situation quite well is for 'black holes', meaning, objects where the mass is inside or at the Schwarzschild radius.

Thanks, Matt. Let's see. The gravitational [self energy](http://www.alternatievewiskunde.nl/cosmology/nasa.htm) $E$ of a body with mass $M$ and the [Schwarzschild radius](https://en.wikipedia.org/wiki/Schwarzschild_radius) $R$ of the same body are respectively given by

$$
E = -\frac{G.M^2}{2R} \quad \mbox{with} \quad R = \frac{2GM}{c^2}
$$

Let Google be your friend and type "energy release quasar". Then we find a [Wikipedia](https://en.wikipedia.org/wiki/Quasar) webpage and the following.
>A quasar (or quasi-stellar object or QSO) is an extremely luminous active galactic nucleus (AGN), in which a supermassive black hole accretes gas and about 10^55 W of energy is released in the form of electromagnetic radiation. They are the highest natural energy release events in a universe. [ .. ]
The huge luminosity of quasars [ .. ] can convert between 5.7% and 32% of the mass of an object into energy, compared to just 0.7% for the p–p chain nuclear fusion process that dominates the energy production in Sun-like stars.

I think it follows from the above math that 25% of the mass of our object is converted into energy, _where the mass is at the Schwarzschild radius_:

$$
E = -\frac{1}{4}Mc^2
$$

**Reply by ExpEarth** [reply]
I don't think they meant that it is the quasar's mass that is converted to energy, but rather the mass of the objects falling onto the accretion disk:
`The huge luminosity of quasars results from the accretion discs of central supermassive black holes, which can convert between 5.7% and 32% of the mass of an object into energy, compared to just 0.7% for the [p–p chain] nuclear fusion] process that dominates the energy production in Sun-like stars.`

In your equation for the self-energy, why do you have 2 in the denominator?

What you really want to show is what the energy content of the radiation _inside_ the quasar is, if you want to compare it to the CMB.

**Reply by HanDeBruijn** [reply]
>I don't think they meant that it is the quasar's mass that is converted to energy, but rather the mass of the objects falling onto the accretion disk

I'm well aware of what they meant to say, Matt. Deletion of the text [ _results from the accretion discs of central supermassive black holes, which_ ] in **my** version of the quote is not at all by coincidence. It's just that I do **not** believe in **accretion disks**. I don't know what it is, but I'm convinced that the energy producing / absorbing mechanism in ACG's must be something quite different.
Fancy animations, **no** observations, as usual:

![AD](https://github.com/a-cosmology-group/acg/assets/135433222/93d8b7fd-5314-4296-89a1-dbd794899c5a)

**Reply by HanDeBruijn** [reply]
@ExpEarth .
>In your equation for the self-energy, why do you have 2 in the denominator?

It's all explained in [Self Energy Field](http://www.alternatievewiskunde.nl/cosmology/nasa.htm), as I have mentioned ad nauseam.
Ah, I see why the concern: that outcome would be contradictory to formula (6) in your paper, where it reads : $L_H = Mc^2 H_0/2$ .

**Reply by HanDeBruijn** [reply]
There are at least two formulations of the [Stefan-Boltzmann law](https://en.wikipedia.org/wiki/Stefan%E2%80%93Boltzmann_law) for (blackbody) heat radiation.
The first one is for the _energy radiated per unit surface area per unit time_: $\sim \sigma T^4$ .
The second one is for the _radiation energy density_ (energy per unit volume): $\sim 4/c.\sigma T^4$ .
I have always been using the second formulation in my attempts to calculate for the CMB.

It seems that @ExpEarth is using instead the second formulation. Then an energy per unit surface area per unit time, times the (radiating) surface area has to be employed for the purpose. I think that's the reason why the concept of a **Hubble luminosity** is predominant in his publications, e.g. [_Does the Hubble redshift flip photons and gravitons?_](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/HubbleRedshift.pdf).
The Hubble luminosity must be the time derivative of the energy : $L_H = dU/dt = - U.H_0$ . It follows that $U = U_0\exp(-H_0.t)$ : energy content is exponentially decreasing with (cosmic) time. Is that correct, Matt?

**Reply by ExpEarth** [reply]
Thanks, Han.  I have noted both uses in my preprint paper in #204 .  You are right that I mostly focus on the first expression involving a luminosity.  However, I am a static guy and do not favour a major evolution over time.  The energy that is given off with the Hubble luminosity is absorbed elsewhere in gravitation, the other side of the coin.  We can summarize this by saying that -_(_dU/dt_)/U_ = (_dE/dt)_/_E_ = +/- _H_ , where we have the cosmic quantities _U_ and _E_ assumed equal in magnitude.

**Reply by HanDeBruijn** [reply]
>The energy that is given off with the Hubble luminosity is absorbed elsewhere in gravitation

_Does the Hubble redshift flip photons and gravitons?_ Yes, with $U$ for radiation energy and $E$ for gravitational energy it is clear from your reply that

$$
U = U_0 e^{-H_0t} \quad \Longleftrightarrow \quad E = E_0 e^{+H_0t}
$$

_where we have the cosmic quantities_ $U$ _and_ $E$ _assumed equal in magnitude._ Thus what we might have is **conservation of energy**. And you say that photon energy is transferred to gravitons.
Another way of speaking may be that the gravitational constant is not a constant but changing with time. See for example formula (4.4) in [Origin of Inertia](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/96836893X-rev.pdf). An idea that has been coined up by Laplace.
Still another way of looking at it is that the [gravitational parameter](https://en.wikipedia.org/wiki/Standard_gravitational_parameter) is changing. Then we can choose between a variable gravitational constant $G$ or a Variable Mass $M$. Both approaches have been tried, the latter by Arp, Narlikar and myself.

$$
G = G_0 e^{+H_0t} \quad \Longleftrightarrow \quad MG = (M.G)_0 e^{+H_0t} = M.G_0 e^{+H_0t} = M_0 e^{+H_0t} G
$$

And there is conservation of mass-energy if

$$
U = \frac{1}{4} M c^2 e^{-H_0t} = \frac{1}{4} \left[M_0 e^{+H_0t}\right] c^2 e^{-H_0t} = \frac{1}{4} M_0 c^2
$$

**Reply by Francois-Zinserling** [reply]
@HanDeBruijn I would consider all reference to $t$ above with $x/c$
I have not ventured much into cosmology but my argument is similar to @ExpEarth , that G is dissipated over distance, with a caviat that the more mass, the more dissipation.

Reason being I don't think G varies with time, but varies with distance; In a sense that everyone sees themselves at the centre of their universe, and in a Machian sense G is reduced **over distance** from afar to local, **not over time**. Difference again that over distance meaning G is conserved locally (as opposed to the idea of still decaying over time). Also, G kind of needs to be a vector for this to work proper, then the gradient of G simply becomes what we know as 'curved space'.

Let's say (hypothetically) that the origin of 'G' is the summation effect of all visible (and invisible) quasars from the distant universe, then the image might look the same from every point in the universe. If you look far away G (directional toward us) seems higher over there than it is local. Same applies to that point though if they were looking back at us, because from our direction their G is also reduced. 

In another practical example we can look at e.g. a disk of a spiral galaxy. From a high G on the edge inward to the centre, G would be much more rapidly reduced than in 'open' space. High G (inward) naturally means higher velocities on the edge.

Since in just about every equation we use, the terms $G*M$ are inseparable, we could argue for the variability of either G or M. In my aether theory, $M = f(aether)$ and also $G = f(aether)$ so it seems our thinking may be somewhat aligned.

**Reply by ExpEarth** [reply]
@HanDeBruijn 
`Another way of speaking may be that the gravitational constant is not a constant but changing with time`
That's not it.  Your equations (and perhaps mine also) give the impression that the quantities _U_ and _E_ change over time.  On the cosmic scale, this is not so.  Electromagnetic energy is converted to gravitational energy and vice versa at the same rate, and so the quantities do not change.  The situation could be more complex in other cases, such as in orbiting bodies.  There is evidence that planets and moons are steadily moving to higher orbits in a way consistent with a decreasing gravitational energy: 

Křížek M, Gueorguiev VG, Maeder A. **An alternative explanation of the orbital expansion of Titan and other bodies in the Solar System**. Gravit Cosmol (2022) 28: 122–132. [https://doi.org/10.1134/S0202289322020086](url)

Vavryčuk V. **Gravitational orbits in the expanding Universe revisited**. Front Astron Space Sci (2023) 10:1071743 [https://doi.org/10.3389/fspas.2023.1071743](url)

Despite these examples, the idea that _G_ changes over time, as Dirac thought, hasn't held up.  Your Variable Mass Hypothesis also has not found support.

@Francois-Zinserling 
`I don't think G varies with time, but varies with distance`
I think _G_ must be constant with distance too.  If it were not, we would see dramatic effects in the galaxies that _JWST_ is now seeing.  The rates of fusion and the galaxy shapes would be different.  But they look pretty normal.  But this is a hard question that requires specific models to answer.  In my shell model, for example, I argue that the universe is a form of black hole and that it and all black holes have a shell-like structure (or ring in rotating black holes).  In my model _G_ is constant in each type of object.

**Reply by HanDeBruijn** [reply]
@ExpEarth .
>Your equations (and perhaps mine also) give the impression that the quantities $U$ and $E$ change over time.

Let's restrict attention to your Hubble luminosity $U$ and forget about $E$. Given **your own** premises, classical calculus offers no alternative for the "impression" that $U$ **must** change over time. That outcome is inevitable. As I wrote:

>The Hubble luminosity must be the time derivative of the energy : $L_H = dU/dt = - U.H_0$ . It follows that $U = U_0\exp(-H_0.t)$ : energy content is exponentially decreasing with (cosmic) time.

Then you write:
>Electromagnetic energy is converted to gravitational energy and vice versa at the same rate, and so the quantities do not change.

Question is: **how** then ? By now I find your objection not much better than: _I don't like the outcome of the math_.

>There is evidence that planets and moons are steadily moving to higher orbits in a way consistent with a decreasing gravitational energy

_Length Contraction is observationally equivalent with the Expansion of Empty Space_. Near the bottom of my [Unified Alternative Cosmology](http://www.alternatievewiskunde.nl/cosmology/wild.htm) page we have:

>4. The Moon is spiraling away from the Earth at a [certain](http://www.alternatievewiskunde.nl/cosmology/trubble.htm#feit) rate per year

Then you write:
>Your Variable Mass Hypothesis also has not found support.

That makes two of us: your [Eggshell Universe](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/preprints202402.0712.v1.pdf) has not found any support either.

**Reply by ExpEarth** [reply]
@HanDeBruijn 

I'm a little puzzled.  I think you are the only one here who has studied my model to any extent, the one given in #204, but it seems even you have not really got the gist of it.  I describe the whole story there of how the two pools of energy exchange with each other !

By the way, my model as described there has changed quite a lot.  There is some new research from GR which strongly supports it, but also forces me to modify it.  It's no longer a thin eggshell, but a really thick shell.

**Comment by HanDeBruijn** [comment]
It **is** an interesting paper. Nevertheless there are quite a few issues in it where I have trouble with.
>Standard cosmology [ .. ] had a less conspicuous precursor already in Newton's questionable treatment of inertia as an effect of
space (not of the matter in it) [ .. ]
Newton’s interpretation of inertia as an effect of space appears to have been the wrong choice that led us into the dark [ .. ]
it fails to explain the empirical fact that the inertia of bodies is proportional to their gravitational attraction. If a theory of gravitation and inertia is to be well founded, it must offer such an explanation [ .. ]
in accordance with [ .. ] Mach’s principle.

I beg to differ. It seems to me that the [Bucket argument - Wikipedia](https://en.wikipedia.org/wiki/Bucket_argument#Detailed_analysis) is difficult to replace by something Machian. One argument is that a bucket in rest and a rotating cosmos quickly gives rise to [star trails](https://github.com/orgs/a-cosmology-group/discussions/215#discussioncomment-9472304) with speeds much higher that the speed of light, which is forbidden in e.g. special relativity.
>[ .. ] we are here not concerned with applied science, i.e. with the art of using science for the solution of practical problems._

I think that it becomes more and more impossible to distinguish "pure" science from the applied.

Typos do not exist in mathematics. There is an **error** in the equations (5) and (6), because of (1) and (2):

$$
D_{exp} = \exp\left(\frac{H}{c}D\right)-1 = \exp\left(\frac{H}{c}\left[\frac{c}{H}\ln(1+z)\right]\right)-1 = z \ne \frac{c}{H}z
$$

But (5) and (6) are crucial in anything that follows, starting with an IMO unmotivated move, this one:
>However, $\Phi$ comes out as finite if the effective $r$ in (11) is not the static distance $D$ but the expanded distance $D_{exp}$ of (5) and (6).

And the rest of the reasoning escapes me.

### Update

Not really of course. The mathematics in the paper is just wrong. Yet the author is on the right track when he writes:
>It is noteworthy that, with the exponential expansion in (5), (12) describes a Yukawa type of potential that approaches the Newtonian potential (11) at small distances. Fields of this type are also obtained if gravity is screened or 'absorbed' in the way imagined by Seeliger [87]

Another way to accomplish the same is to adopt the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs), as is demonstrated at the webpage named [_Seeliger's Paradox_](http://www.alternatievewiskunde.nl/cosmology/india.htm).
Anyway, a **correct derivation** is given in the book [_Origin of Inertia_](http://www.alternatievewiskunde.nl/cosmology/Ghosh.htm#PE) by Amitabha Ghosh, which is mimicked at the webpage with the same name.

But let's quote from the paper again:
>In a universe in which matter is homogeneously and isotropically distributed in static space, the potential corresponding to (12) can be calculated by integrating the contributions from shells of thickness dr at distance $r=0$ to $\infty$.

Yes, and according to _Origin of Inertia_ we then get the surprisingly simple result that the potential energy of any particle due to its bare presence in the universe is $\Phi=-m_0c^2$ and hence _the total energy content of the universe is nil._

### Edit

Let's compare the right way with a wrong way of doing the same. The latter is more or less similar to the approach in the apostate's paper (though maybe a bit better).

$$
\Phi = \int_0^\infty \left[ \int_r^\infty \frac{G.\rho.e^{-H/c.x}}{x^2}dx \right] 4\pi r^2 dr \quad \neq \quad \int_0^\infty \left[ \int_r^\infty \frac{G.\rho}{x^2}dx \right] 4\pi r^2 e^{-H/c.r} dr
$$

If $r^2/r=r$ instead of the spurious $r^2/r = r^2/(c.z/H)$ then the (still incorrect) right hand side evaluates into

$$
\Phi \ne \int_0^\infty \frac{-G.\rho}{r} 4\pi.r.e^{-H/c.r} dr = -G.\rho.4\pi \left(\frac{c}{H}\right)^2 \int_0^\infty \xi.e^{-\xi} d\xi = -4\pi.G.\rho \left(\frac{c}{H}\right)^2
$$

The remaining integral has become $=1$ because of $\int \xi . e^{-\xi} d\xi = -e^{-\xi}(1+\xi)$ .
According to the paper we have $H^2 = 8\pi/3.G.\rho$, so we have $4\pi.G.\rho = 3/2.H^2$. And thus, finally:

$$
m.\Phi \ne -\frac{3}{2} m.c^2
$$

This means that the (wrong) potential energy of an arbitrary particle due to its bare presence in the universe is rather nearby the correct outcome $-mc^2$, that is: a factor $3/2$ apart from it.
